tests/ios/Podfile.lock

```diff
@@ -1091,25 +1091,25 @@ SPEC CHECKSUMS:
   React-RCTText: fae545b10cfdb3d247c36c56f61a94cfd6dba41d
   React-RCTVibration: 4356114dbcba4ce66991096e51a66e61eda51256
   ReactCommon: ed4e11d27609d571e7eee8b65548efc191116eb3
-  RNFBAdMob: 8008db9b4fb348ee1bcd707f066fa760873ea8da
-  RNFBAnalytics: 354446ab54bd1e094d82c33d1b232f3c70a4a90e
-  RNFBApp: c62b6f3442edf56ed004bd77240347647475ec9e
-  RNFBAuth: 6e8d1412db2ee67db4c8514ce95ee458195c6e70
-  RNFBCrashlytics: 7da01fba0dd592b3c2d9ee1b088bd747df36d458
-  RNFBDatabase: c8d58137acc264075ac1f637f9944e6d97bbf1af
-  RNFBDynamicLinks: 2227e2731a457e51c2ceca4dbccda1148a6a63c7
-  RNFBFirestore: 455c682df9e0e323177c42383f6205ae39533b15
-  RNFBFunctions: 4ca7ed9a30f5791f418f714346c301c31b8cb913
-  RNFBIid: 0bb305b1accc30ed81b94acb4d1fbb1cdb9d97af
-  RNFBInAppMessaging: 13d0d207fdd8c24a07857b2c8befdc4848c8a7af
-  RNFBMessaging: bb1343d10652c5dd9c18124fd2f0246881203274
-  RNFBMLNaturalLanguage: 919804054a99e98817fe6cd63904a58f26d45111
-  RNFBMLVision: ff4bbc32b67f80064f1510eb0275ffb3ac829f88
-  RNFBPerf: f95655c914cc4b1c3c829b68465fbe59dfb5e7b3
-  RNFBRemoteConfig: 134e3fdc55d6d5aee69d9c083c13f8694b6510c3
-  RNFBStorage: 442afb7abb614ee44ecbdd8a7bec42e6d4210302
+  RNFBAdMob: 809f648889201406d333bc28a84bbf3294491f00
+  RNFBAnalytics: 159651d6eae3c85db38ba5d694d8c6c46fd3883c
+  RNFBApp: e0fc0113eecc07f440f17639c9b7c59ea90bc583
+  RNFBAuth: 16207757fa69ad50ec8ca04964f59cd560979294
+  RNFBCrashlytics: c85d01c3fb3a4cc1e762facb9d4ad26b95f7f9dc
+  RNFBDatabase: 6c01157824702f4fc1cedf9b4b95e9f3154cfbf1
+  RNFBDynamicLinks: 067d7419d8daf58b61faa70834b051410d5f6d4b
+  RNFBFirestore: 64986e129f4980e73b6e510684286d986367bef6
+  RNFBFunctions: ae7a279090e902cdf4da7890d64f31a0e4e2a825
+  RNFBIid: f40ac75229f8bb86cc6dd0c71b450e7df693f8f3
+  RNFBInAppMessaging: dda5e571edd579042fa1d68a685012daa871a3f6
+  RNFBMessaging: 1d2a6a249cf6b93bed1721befc42650c45615112
+  RNFBMLNaturalLanguage: 3662457b2e95b857bb9289d250b0a10bc10aca84
+  RNFBMLVision: c2547c24b59298ebe4b90a2025600d60a6929930
+  RNFBPerf: 0c08e45726f7a19487b79cef3d12ee7e917c8b7a
+  RNFBRemoteConfig: 85df9235f46c20e293257b6e481412ac585e2966
+  RNFBStorage: 72404d4977261c0d7060e87c3d0cf7f7e060c6a3
   Yoga: 3ebccbdd559724312790e7742142d062476b698e
 
-PODFILE CHECKSUM: 555dacf747a9f802fb0d5efa4c6f558ca6129c4d
+PODFILE CHECKSUM: 2b670e97319c5057f900292e8500c4c38d83aa3c
 
-COCOAPODS: 1.10.0.rc.1
+COCOAPODS: 1.9.3
```

Comment:
This is unrelated just a tangent that 1.10 is out as general release now.

Reply:
updated 😄